ui/components/ui/token-list-display/token-list-display.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { isEqual } from 'lodash';
+
+import { getShouldHideZeroBalanceTokens } from '../../../selectors';
+import { useTokenTracker } from '../../../hooks/useTokenTracker';
+import Identicon from '../identicon';
+import TokenBalance from '../token-balance';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getTokens } from '../../../ducks/metamask/metamask';
+
+export default function TokenListDisplay(props) {
+  const t = useI18nContext();
+  const shouldHideZeroBalanceTokens = useSelector(
+    getShouldHideZeroBalanceTokens,
+  );
+
+  const { clickHandler } = props;
```

Comment:
[REQ] If we could destructure this in the function signature that'd be preferable.

Reply:
The main reason being we already do it in most places and we should try and find a cohesive style guide so that each engineers preferences don't lead to many different patterns.I'd love to hear more about the react team's recommendations and why if you can find me some links/material -- and we can discuss making this the standard moving forward. But as of now i'd like most react components we write to look as close to the same style as possible.